CppCoreGuidelines.md

```diff
@@ -19135,7 +19144,7 @@ to name their own UDLs `operator""_x` - they will not collide with the standard
 
 ##### Enforcement
 
-Flag `using namespace` at global scope in a header file.
+Flag `using namespace` at global scope, or namespace declrations that only include other  in a header file.
```

Comment:
typo "declrations"

seems like there's supposed to be another word between "other" and "in". presumably "namespaces"